### PR TITLE
Add standards subcategory including ASyncAPI and OpenAPI

### LINKED ISSUE #47: Modify the adopters subcategories
As discussed in the group call lets refine the adopters subcategories to have the following:
* Standards: moving into it W3C Internet of Things and adding 2 news adopters: AsyncAPI and OpenAPI.
* Adopters: The rest of adopters.

### PR DESCRIPTION
**Pull request by benjagm**
Add standards subcategory including ASyncAPI and OpenAPI

**Issue Number:**

-  Closes #47 

**Screenshots/videos:**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

### CODE REVIEW

**Review by DarhkVoyd — COMMENTED** [review]
Shouldn't we also update `settings.yml` to include the updated categories.

**Comment by benjagm** [comment]
> Shouldn't we also update `settings.yml` to include the updated categories.

The Categories/Subcategories in the settings.yml file is used for excluding elements from the view:

````
# Categories (optional)
#
# Categories information is read from the `landscape.yml` data file. The way
# categories are displayed in the web application is computed dynamically based
# on the number of categories and subcategories, as well as the number of items
# on each. Sometimes, however, we may want subcategories to be displayed in a
# specific order within a category (this happens often in the Members category).
# In those cases, it is possible to define that order by overriding a category.
#
# This option can also be used to not display one or more subcategories in the
# landscape. To achieve that, we only need to exclude the subcategory from the
# subcategories list when overriding a category.
#
# categories:
#   - name: <CATEGORY_NAME>
#     subcategories:
#       - <SUBCATEGORY1_NAME>
#       - <SUBCATEGORY2_NAME>
````

**Review by DarhkVoyd — APPROVED** [review]
Apologies, I confused these with this:
```
# Groups (optional)
#
# In some cases, specially when a landscape contains lots of items, it may be
# interesting to organize them in groups. Each group will be displayed on a
# different tab in the landscape web application. Each entry must contain the
# name of the group (it will be displayed as is) and a list with the categories
# (as defined in the landscape.yml data file) that will be part of this group.
#
# groups:
#   - name: <GROUP_NAME>
#     categories:
#       - <CATEGORY1_NAME>
#       - <CATEGORY2_NAME>
#
groups:
  - name: JSON Schema Ecosystem
    categories:
      - Adopters
      - Sponsors
      - Tools
  - name: Adopters
    categories:
      - Adopters
  - name: Sponsors
    categories:
      - Sponsors
  - name: Tools
    categories:
      - Tools
```

File: data.yml [hunk 1]
```diff
@@ -2,6 +2,32 @@ landscape:
...
+      - subcategory:
+        name: Standards
+        items:
+          - item:
+            name: W3C Web of Things
+            homepage_url: https://www.w3.org/WoT/
+            logo: w3cwot.svg
+            crunchbase: https://www.crunchbase.com/organization/world-wide-web-consortium
+            project_org: https://github.com/w3c
+            description: The World Wide Web Consortium (W3C) develops interoperable technologies (specifications, guidelines, software.
```
Comment (DhairyaMajmudar) [inline data.yml:14]: Here pls. add closing bracket `)`

**Comment by DhairyaMajmudar** [comment]
One small comment rest everything is looking good : )

**Comment by benjagm** [comment]
> One small comment rest everything is looking good : )

I fixed it with the last push. Thanks! Good catch.

**Pull request merged**